apps/emqx/test/emqx_persistent_session_ds_SUITE.erl

```diff
@@ -290,7 +290,7 @@ t_storage_generations(Config) ->
             ok = emqtt:puback(Sub, PI2),
             ok = emqtt:puback(Sub, PI3),
             ok = emqtt:puback(Sub, PI4),
-            [#{packet_id := PI5}] = emqx_common_test_helpers:wait_publishes(1, 5_000)
+            [#{packet_id := _PI5}] = emqx_common_test_helpers:wait_publishes(1, 5_000)
```

Comment:
I fixed this on my branch as well (and changed the tests), so perhaps leave it out to avoid conflict

Reply:
removed